Codeinwp/neve
# Post navigation & Archive description

### Summary


- Next/Previous links for Single Post
- Display description under the title on archive page

### Will affect visual aspect of the product

YES

### Screenshots 

### Test instructions


#307
- Open a single post page
- Check that Next/Previous links appear
- Click on them and check if another post page is opened

#317
- Add description to category/author/tag
- Check that description shows up under the title (between the title and the border)


Closes #307 #317
